### ISSUE TITLE
Default color maps makes points invisible

### ISSUE THREAD

**Issue by ivirshup** [pyviz/hvplot#178]
Points with the maximum value are invisible by default for scalar valued color maps. I think this is bad default behavior. Here's an example:

```python
import pandas as pd
import numpy as np
import hvplot.pandas
from itertools import repeat

d = pd.DataFrame({"x": np.arange(10), "y": np.arange(10), "c": np.ones(10)})
d.loc[5, "c"] = 0

d.hvplot.scatter("x", "y", c="c")
```
<img width="788" alt="image" src="https://user-images.githubusercontent.com/8238804/53612029-c1d59500-3c24-11e9-8ca8-3ff66a9d2580.png">

**Comment by philippjfr** [comment]
Definitely agreed on that, related discussion in https://github.com/pyviz/holoviews/issues/3500

**Comment by philippjfr** [comment]
I think we should change the default colormap in hvPlot independently from HoloViews.

**Comment by jbednar** [comment]
See discussion in https://github.com/pyviz/holoviews/issues/3500 before deciding...

**Comment by jsignell** [comment]
Just to note that matplotlib has switched the colormap direction so that dark colors indicate high values by default, so I think it would be relatively safe for us to move to `kbc_r`.

```python
ax = d.plot.scatter("x", "y", c="c")
d.hvplot.scatter("x", "y", c="c", cmap='kbc_r', width=400)
```
<img width="959" alt="Screen Shot 2019-07-24 at 1 48 22 PM" src="https://user-images.githubusercontent.com/4806877/61816235-f4716b80-ae19-11e9-8d6f-ec765405513d.png">

**Comment by jbednar** [comment]
Oh, that's great news!  I think that's really helpful for the future of science, which appears to be occurring on a white background. :-) Can you link to the PR where that change was made (here and in https://github.com/pyviz/holoviews/issues/3500), so that we have the surrounding discussion for context?

**Comment by jsignell** [comment]
We could even check for if `bgcolor` is set and flip the cmap if it's `black`.

**Comment by jbednar** [comment]
Sure; that's what my old code for running simulations did (circa 1995...)

**Comment by jsignell** [comment]
I can't quite find the PR or even a blog post announcing the change.

**Comment by jsignell** [comment]
closed in #258
